modules/organization-admin/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "admin_account_id" {
```

Comment:
Suggest maintaining consistency with the `delegated_administrator_account_id` terminology we're going with in guardduty?